WebDevStudios/nextjs-wordpress-starter
# Fix/split lazyblocks

Closes N/A

### Link

https://vercel.com/webdevstudios/nextjs-wordpress-starter/dbj2dsgck?redirect=1

### Description

Split the blocks folder into `gutenberg/` and `lazyblocks/` directories for better organization

### Screenshot

![ 2021-02-02 at 11 02 00 ](https://user-images.githubusercontent.com/12365909/106626825-14710780-6546-11eb-80e6-2a796fabeea9.png)

### Verification

How will a stakeholder test this?

1. Build the app
2. Make sure there are no errors
